code/obj/item/device/nuclear_op_items.dm

```diff
@@ -0,0 +1,919 @@
+
+/obj/item/remote/nuke_summon_remote/proc/try_to_find_the_nuke()
+	if(ticker.mode.type == /datum/game_mode/nuclear)
+		var/datum/game_mode/nuclear/mode = ticker.mode
+		the_bomb = mode.the_bomb
+	if(isnull(the_bomb))
+		for_by_tcl(nuke, /obj/machinery/nuclearbomb)
+			the_bomb = nuke
+			break
+
+/obj/item/remote/nuke_summon_remote/proc/tele_the_bomb(mob/user as mob)
+	showswirl(the_bomb)
+	the_bomb.set_loc(get_turf(src))
+	showswirl(src)
+	src.visible_message("<span class='alert'>[user] has summoned the [the_bomb]!</span>")
+	src.charges -= 1
+	playsound(src.loc, use_sound, 70, 1)
+
+////////// Reinforcement Summon Beacon //////////
+/obj/item/remote/reinforcement_beacon
+	name = "Reinforcement Beacon"
+	icon = 'icons/obj/items/device.dmi'
+	desc = "A handheld beacon that allows you to call a Syndicate gunbot to the user's current location."
+	icon_state = "beacon" //replace later
+	item_state = "electronic"
+	density = FALSE
+	anchored = FALSE
+	w_class = W_CLASS_SMALL
+	var/uses = 1
+	var/ghost_confirmation_delay = 30 SECONDS
+
+	New()
+		..()
+		desc = "A handheld beacon that allows you to call a Syndicate gunbot to the user's current location. It has [src.uses] charge left."
+
+/obj/item/remote/reinforcement_beacon/attack_self(mob/user as mob)
+	if(!isonstationz(user.z))
+		boutput(user, "<span class='alert'>The [src] can't be used here, try again on station!</span>")
+		return
+
+	if(uses >= 1)
+		uses -= 1
+		boutput(user, "<span class='alert'>You activate the [src], before setting it down on the ground.</span>")
+		src.force_drop(user)
+		src.anchored = TRUE
+		sleep(1 SECOND)
+		src.visible_message("<span class='alert'>The [src] beeps, before locking itself to the ground.</span>")
+		src.desc = "A handheld beacon that allows you to call a Syndicate gunbot to the user's current location. It seems to currently be transmitting something."
+		sleep(5 SECONDS)
+		var/list/text_messages = list()
+		text_messages.Add("Would you like to respawn as a nuclear operative gunbot? You may be randomly selected from the list of candidates.")
+		text_messages.Add("You are eligible to be respawned as a nuclear operative gunbot. You have [src.ghost_confirmation_delay / 10] seconds to respond to the offer.")
+		text_messages.Add("You have been added to the list of eligible candidates. Please wait for the game to choose, good luck!")
+
+		// The proc takes care of all the necessary work (job-banned etc checks, confirmation delay).
+		message_admins("Sending Syndicate Reinforcement offer to eligible ghosts. They have [src.ghost_confirmation_delay / 10] seconds to respond.")
+		var/list/datum/mind/candidates = dead_player_list(1, src.ghost_confirmation_delay, text_messages, allow_dead_antags = 1)
+		if(!length(candidates))
+			src.visible_message("<span class='alert'>The [src] buzzes, before unbolting itself from the ground. There seems to be no reinforcements available currently.</span>")
```

Comment:
Would suck on lowpop

Reply:
I don't really know how to help with this, you can already refund it by hitting the uplink with it, and nukeops can just kill people to get more ghosts

Reply:
Fair enough.